src/main/java/com/findmyrecycling/fmrenterprise/service/IPhotoService.java

```diff
@@ -14,16 +14,16 @@ public interface IPhotoService {
      * Display a photo on the screen
      * @param photo is a photo to be displayed
      */
-    void display(Photo photo);
+    void display(Photo photo) throws Exception;
```

Comment:
This isn't necessarily adding value.
- Methods always throw runtime exceptions, by default.  Is there a checked exception here that should be thrown?  If so, be specific and state what checked exception will be thrown.
- Does the implementing class throw an exception?  If so, I'd expect to see changes to that class as well, as part of this commit.  If not, this method signature change is misleading.